toString.js

```diff
@@ -0,0 +1,42 @@
+const LINES = {
+  r: '└──', //right
+  t: '├──', //T shape
+  v: '│  '  //vertical
+}
+
+function toString() {
+  console.log('Fastify Instance toString')
+  /* Print hooks */
+  console.log(`\nHooks`)
```

Comment:
This should be "Top level Hooks". I don't think you can see hooks that are wrapped within a plugin.